app/backend/api_wrappers/hugging_face.py

```diff
@@ -74,6 +75,11 @@ async def chat_completion(
             top_p=top_p,
         )
 
+    async def text_classification(
+        self, text: str, model: Optional[str] = None
```

Comment:
That's my mistake, I was also planning to make the PromptProtection instance to store its own hugging face client, which would allow to set the default model to use, then this would be optional.

Reply:
Fixed